### ISSUE TITLE
Discord RPC Buttons

### ISSUE THREAD

**Issue by Rayy777** [niveshbirangal/discord-rpc#12]
discord now allows you to add a maximum of 2 buttons with links for you rich presence, this rpc doesnt seem to be able to do that currently

**Comment by niveshbirangal** [comment]
Can you show me an example? (Screenshot)

**Comment by PblSoftWare** [comment]
What I have to do to put a button with a link?

**Comment by niveshbirangal** [comment]
I haven't seen a custom rich presence with the button I guess. if you have reference to another custom presence that has a button I can implement it in this one also.

**Comment by PblSoftWare** [comment]
it could be nice

**Comment by PblSoftWare** [comment]
I mean I would use it

**Comment by PblSoftWare** [comment]
In discord presence does exist two types of buttons:
- Join something
- And a just a link
The point is that both works the same, but they look different and that the one of joining stuff also have a variable for the ppl who has joined and that stuff

**Comment by niveshbirangal** [comment]
@PblSoftWare Yes, you can. For that, you need to integrate your current app with the Discord GameSDK and then you can set it. 
You can find out more here - https://discord.com/developers/docs/rich-presence/how-to

**Comment by PblSoftWare** [comment]
oh, thanks
